satpy/resample.py

```diff
@@ -874,7 +882,8 @@ def resample_dataset(dataset, destination_area, **kwargs):
 
         return dataset
 
-    new_data = resample(source_area, dataset, destination_area, **kwargs)
+    fill_value = kwargs.pop('fill_value', get_fill_value(dataset))
+    new_data = resample(source_area, dataset, destination_area, fill_value=fill_value, **kwargs)
```

Comment:
Should this do `kwargs.setdefault('fill_value', fill_value)` instead so someone/something can pass `fill_value` to `resample_dataset`?